WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -125,6 +125,7 @@ private struct ProductsSection: View {
                 }, content: {
                     AddProductToOrder(isPresented: $showAddProduct, viewModel: viewModel.addProductViewModel)
                         .onDisappear {
+                            viewModel.addProductViewModel.searchTerm = ""
```

Comment:
Do you think this could be `viewModel.cleanSearchTerm()`? That way we can unit test it!

Reply:
Definitely! I was going to do it that way and then thought maybe it was overkill. I'll add it. :)